tests.py

```diff
@@ -743,7 +743,11 @@ def _run(self, *command):
         if rc:
             print(' '.join(command))
             print(stdout)
-            raise subprocess.CalledProcessError(rc, command[0], output=stdout)
+            try:
+                raise subprocess.CalledProcessError(rc, command[0], output=stdout)
+            except TypeError:
+                # BBB Python 2.6
+                raise subprocess.CalledProcessError(rc, command[0])
```

Comment:
Oh, this is test code!  Ok, I won't insist on a separate test for testing error handling paths that occur during test failures.